### ISSUE TITLE
[Feature Request]: "Looped Replay" button in Play Controls area. 

### ISSUE THREAD

**Issue by my-life-is-confetti** [u8sand/Baka-MPlayer#281]
Does lib-mpv or mpv provide for this functionality? If so, I would like it added to Baka-MPlayer.

I am on Windows 10 Pro 64-bit if that matters.

**Comment by u8sand** [comment]
I was under the impression we had this feature; i believe it's one or the
playlist controls.

**Comment by my-life-is-confetti** [comment]
Hi Daniel!,

It's been a while; this is Leo Henslee III, "ConundrumAdInfinitum", one of your Windows 10 x64 users.

See that you're working on Mochi Player, I'm curious how the progress is coming? It'd be great to see Baka- or Mochi- get packaged for the Microsoft Store. I'm sure one of the Bridges (Centennial? or the Xamarin tool; which works for Linux code, I believe).

If there is anything I could do for the Project, I'd be interested in volunteering. I know my way around the Windows file system but I don't code. I could definitely test, or do documentation. I'm a decent Wordsmith.

I'll take a peek. What I was hoping for was a UI improvement where a Replay and Replay Continuous button(s) could be added next to the Play Control buttons? Would that be possible?

Leo H
my.life.is.confetti

________________________________

**Comment by my-life-is-confetti** [comment]
Yes. Shuffle, and Repeat are both in the Playback Menu. Is there a chance they could get added as buttons to the Left of the Playback Buttons on the Main UI?


Best,
Leo H
my.life.is.confetti
________________________________

**Comment by my-life-is-confetti** [comment]
OH! also, I usually control Baka- via the 10-key via custom KB Shortcuts... Is there the chance of binding Shuffle and Repeat to specific Keys? Couldn't find them in the "Settings>Preferences>Key Bindings" menu...

Best,
Leo H

my.life.is.confetti
________________________________
